### PR TITLE
MIG] sale_order_type: Migration to 11.0

### PR DESCRIPTION
**Pull request by chienandalu**
Sales Order Type
================

This module adds a typology for the sales orders. In each different type, you
can define, invoicing and refunding journal, a warehouse, a sequence,
the shipping policy, the invoicing policy, a payment term, a pricelist
and an incoterm.

You can see sale types as lines of business.

You are able to select a sales order type by partner so that when you add a
partner to a sales order it will get the related info to it.

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@chienandalu there's something strange in your extraction method, as there are other modules involved in the PR. Commit 6b0779024dfcb52d27c622443d67949b5f01d0f7 is one of the affected.

File: sale_order_type/README.rst [hunk 1]
```diff
@@ -57,6 +57,9 @@ Contributors
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
 * Samuel Lefever <sam@niboo.be>
 * Pierre Faniel <pierre@niboo.be>
+* David Vidal <david.vidal@tecnativa.com>
```
Comment (pedrobaeza) [inline sale_order_type/README.rst]: I'm above with my old email. Group them and put company in first level

File: sale_order_type/__manifest__.py [hunk 2]
```diff
@@ -4,11 +4,12 @@
...
+# Copyright 2018 David Vidal <david.vidal@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Sale Order Type",
-    "version": "10.0.1.0.0",
+    "version": "11.0.1.0.0",
     "category": "Sales Management",
     "author": "Grupo Vermon,"
               "AvanzOSC,"
```
Comment (pedrobaeza) [inline sale_order_type/__manifest__.py]: Depends on sale_management for seeing the menus

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by chienandalu** [comment]
@pedrobaeza  I passed 2to3 in the repo folder instead of the module one and there were some undesired changes.

File: sale_order_type/README.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (pedrobaeza) [inline sale_order_type/README.rst]: Update this part to latest template, and add the same title line on the top part of the title.

File: sale_order_type/README.rst [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+You are able to select a sales order type by partner so that when you add a
+partner to a sales order it will get the related info to it.
+
+Configuration
+=============
+
+To configure Sale Order Types you need to:
+
+1. Go to **Sales > Configuration > Sales Orders Types**
```
Comment (pedrobaeza) [inline sale_order_type/README.rst]: Use `#.`

File: sale_order_type/README.rst [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+2. Create a new sale order type with all the settings you want
+
+Usage
+=====
+
+* Go to **Sales > Sales Orders** and create a new sale order. Select the new
```
Comment (pedrobaeza) [inline sale_order_type/README.rst]: Use `#.`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please squash a bit the commit history by logical set/author for cleaning up

File: sale_order_type/README.rst [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (JayVora-SerpentCS) [inline sale_order_type/README.rst]: https!

**Comment by chienandalu** [comment]
commits squashed and comments attended

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@JayVora-SerpentCS can you please update your review?

**Review by JayVora-SerpentCS — APPROVED** [review]

**Review by andhit-r — APPROVED** [review]

**Pull request merged**